recipes/matlab-mode

```diff
@@ -1,9 +1,9 @@
 (matlab-mode
  :fetcher git
- :url "https://git.code.sf.net/p/matlab-emacs/src"
+ :url "https://github.com/mathworks/Emacs-MATLAB-Mode.git"
  :files ("*.el" "*.m" ("toolbox" "toolbox/*.m")
          ("toolbox/+emacs" "toolbox/+emacs/*.m")
          ("toolbox/+emacs/@Breakpoints" "toolbox/+emacs/@Breakpoints/*.m")
          ("toolbox/+emacs/@EmacsServer" "toolbox/+emacs/@EmacsServer/*.m")
          ("toolbox/+emacs/@Stack" "toolbox/+emacs/@Stack/*.m")
-         ("templates" "templates/*.srt")))
+         ("bin" "bin/*.sh")))
```

Comment:
It looks like this could also use `:defaults` instead of `*.el` yielding -
```elisp
(matlab-mode
 :fetcher github
 :repo "mathworks/Emacs-MATLAB-Mode"
 :files (:defaults "*.m" ("toolbox" "toolbox/*.m")
         ("toolbox/+emacs" "toolbox/+emacs/*.m")
         ("toolbox/+emacs/@Breakpoints" "toolbox/+emacs/@Breakpoints/*.m")
         ("toolbox/+emacs/@EmacsServer" "toolbox/+emacs/@EmacsServer/*.m")
         ("toolbox/+emacs/@Stack" "toolbox/+emacs/@Stack/*.m")
         ("bin" "bin/*.sh")))
```

Reply:
I took your recommendation, changed the recipe, and amended (as Jonas recommended) the change to my first commit on my branch matlab.